datasets/imagenet/README.md

```diff
@@ -0,0 +1,1221 @@
+
+#### Who are the source language producers?
+
+WordNet synsets further quality controlled by human annotators. The images are from Flickr.
+
+### Annotations
+
+#### Annotation process
+
+The annotation process of collecting ImageNet for image classification task is a three step process.
+
+1. Defining the 1000 object categories for the image classification task. These categories have evolved over the years.
+1. Collecting the candidate image for these object categories using a search engine.
+1. Quality control on the candidate images by using human annotators on Amazon Mechanical Turk (AMT) to make sure the image has the synset it was collected for.
+
+See the section 3.1 in [1](https://arxiv.org/abs/1409.0575) for more details on data collection procedure and [2](https://ieeexplore.ieee.org/abstract/document/5206848) for general information on ImageNet.
+
+#### Who are the annotators?
+
+Images are automatically fetched from an image search engine based on the synsets and filtered using human annotators on Amazon Mechanical Turk. See [1](https://arxiv.org/abs/1409.0575) for more details.
+
+### Personal and Sensitive Information
+
+See the papers: [1](https://arxiv.org/abs/1409.0575) [2](https://ieeexplore.ieee.org/abstract/document/5206848).
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+ImageNet dataset has been very crucial in advancement of deep learning technology as being the standard benchmark for vision models. The dataset aims to probe models on their understanding of the objects and has successfully shouldered the responsibility of holding through the test of time for that purpose. ImageNet is still one of the major datasets on which models are evaluated for their generalization in computer vision capabilities as the field moves towards self-supervised algorithms. Please see the future section in [1](https://arxiv.org/abs/1409.0575).
+
+### Discussion of Biases
+
+1. A [study](https://image-net.org/update-sep-17-2019.php) of the history of the multiple layers (taxonomy, object classes and labeling) of ImageNet and WordNet in 2019 described how bias is deeply embedded in most classification approaches for of all sorts of images.
+1. A [study](https://arxiv.org/abs/1811.12231) has also shown that ImageNet trained models are biased towards texture rather than shapes which in contrast with how humans do object classification. Increasing the shape bias improves the accuracy and robustness.
+1. Another [study](https://arxiv.org/abs/2109.13228) more potential issues and biases with the ImageNet dataset and provides an alternative benchmark on top. The data collected contains humans without their consent.
+1. ImageNet data with face obfuscation is also provided at [this link](https://image-net.org/face-obfuscation/)
+
```

Comment:
Maybe I missed something, but maybe we should add that the most recent version of ImageNet has "filtered 2,702 synsets in the "person" subtree that may cause problematic behaviors of the model." (see https://www.image-net.org/filtering-and-balancing/)

Reply:
Added these.